cypress/integration/block-text.js

```diff
@@ -27,6 +27,36 @@ if (Cypress.env('API') !== 'guillotina') {
         .should('have.attr', 'href')
         .and('include', 'https://google.com');
     });
+
+    it('As a site administrator I can add a new line to a text block using shift + enter', function() {
+      cy.visit('/');
+      cy.get('#toolbar-add').click();
+      cy.get('#toolbar-add-document').click();
+      cy.get('.documentFirstHeading > .public-DraftStyleDefault-block')
+        .type('This is a page')
+        .get('.documentFirstHeading span[data-text]')
+        .contains('This is a page');
```

Comment:
This block shouldn't be necessary since the test setup already opens a page in edit mode.

Reply:
@tisto don't understand